grails-app/domain/org/pih/warehouse/inventory/InventoryItem.groovy

```diff
@@ -132,4 +132,7 @@ class InventoryItem implements Serializable {
         return [product?.id]
     }
 
+    def getIsAutopickable() {
```

Comment:
Also I always forget the convention when it comes to boolean properties, but I think you can just add "autoPickable" to the transients and isAutoPickable() as the method name.

Reply:
https://stackoverflow.com/questions/5322648/for-a-boolean-field-what-is-the-naming-convention-for-its-getter-setter